utils/s2n_safety_macros.h

```diff
@@ -0,0 +1,415 @@
+
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#pragma once
+
+/**
+ * DO NOT DIRECTLY MODIFY THIS FILE:
+ *
+ * The code in this file is generated from scripts/s2n_safety_macros.py and any modifications
+ * should be in there.
```

Comment:
Can we enforce this in any way?

Reply:
Yes. We can add a GHA that runs and compares the output of the script to what's there. I can add that on the tracking issue.

Reply:
I added an item to capture this in https://github.com/awslabs/s2n/issues/2425